phpunit/functional/Glpi/Form/Condition/ConditionHandler/MultipleChoiceFromValuesConditionHandlerTest.php

```diff
@@ -0,0 +1,294 @@
+            'submitted_answer'    => ["option_a"],
+            'expected_result'     => true,
+            'question_extra_data' => $extra_data
+        ];
+        yield "Not equals check - case 5 for $type" => [
+            'question_type'       => $type,
+            'condition_operator'  => ValueOperator::NOT_EQUALS,
+            'condition_value'     => ["option_a", "option_c"],
+            'submitted_answer'    => ["option_c", "option_a"],
+            'expected_result'     => false,
+            'question_extra_data' => $extra_data
+        ];
+        yield "Not equals check - case 6 for $type" => [
+            'question_type'       => $type,
+            'condition_operator'  => ValueOperator::NOT_EQUALS,
+            'condition_value'     => ["option_a", "option_c"],
+            'submitted_answer'    => ["option_a", "option_c"],
+            'expected_result'     => false,
+            'question_extra_data' => $extra_data
+        ];
+        yield "Not equals check - case 7 for $type" => [
+            'question_type'       => $type,
+            'condition_operator'  => ValueOperator::NOT_EQUALS,
+            'condition_value'     => ["option_a", "option_c"],
+            'submitted_answer'    => ["option_a", "option_c", "option_b"],
+            'expected_result'     => true,
+            'question_extra_data' => $extra_data
+        ];
+
+        // Test with the CONTAINS operator
+        yield "Contains check - case 1 for $type" => [
+            'question_type'       => $type,
+            'condition_operator'  => ValueOperator::CONTAINS,
+            'condition_value'     => ["option_c"],
+            'submitted_answer'    => ["option_b"],
+            'expected_result'     => false,
+            'question_extra_data' => $extra_data
+        ];
+        yield "Contains check - case 2 for $type" => [
+            'question_type'       => $type,
+            'condition_operator'  => ValueOperator::CONTAINS,
+            'condition_value'     => ["option_c"],
+            'submitted_answer'    => ["option_c"],
+            'expected_result'     => true,
+            'question_extra_data' => $extra_data
+        ];
+        yield "Contains check - case 3 for $type" => [
+            'question_type'       => $type,
+            'condition_operator'  => ValueOperator::CONTAINS,
+            'condition_value'     => ["option_a", "option_c"],
+            'submitted_answer'    => ["option_c"],
+            'expected_result'     => true,
+            'question_extra_data' => $extra_data
+        ];
+        yield "Contains check - case 4 for $type" => [
+            'question_type'       => $type,
+            'condition_operator'  => ValueOperator::CONTAINS,
+            'condition_value'     => ["option_a", "option_c"],
+            'submitted_answer'    => ["option_a"],
+            'expected_result'     => true,
+            'question_extra_data' => $extra_data
+        ];
```

Comment:
I wonder if this corresponds to the behaviour expected by the form administrators. Indeed, they may expect all the condition values to be found in the answer.